js/components/accessibilityControls.jsx

```diff
@@ -166,34 +170,19 @@ AccessibilityControls.prototype = {
     this.controller.moveVrToDirection(false, keyDirectionMap[charCode]); // stop rotation if isKeyDown === false or prevent prev rotation if press a button (isKeyDown === true)
 
     if (isKeyDown === true) {
-      var newBtn = true;
-      for (var j = this.prevKeyPressedArr.length - 1; j >= 0; j--) {
-        if (this.prevKeyPressedArr[j] === charCode) {
-          newBtn = false; // there is extra pressDown for pressed btn in chrome (windows) and safari if to change to fullscreen mode
-          break; // we do not need to add charCode to prevKeyPressedArr in this case
-        }
-      }
+      const newBtn = this.prevKeyPressedArr.some(address => address === charCode);
       if (newBtn) {
         this.prevKeyPressedArr.push(charCode);
       }
     } else {
-      // if button is up, remove it from this.prevKeyPressedArr
-      var inPrevKeyPressedArrIndex = -1;
-      // check if button code is already in list of pressed buttons (this.prevKeyPressedArr)
-      // if code is in the array return index of the code
-      for (var i = this.prevKeyPressedArr.length - 1; i >= 0; i--) {
-        if (this.prevKeyPressedArr[i] === charCode) {
-          inPrevKeyPressedArrIndex = i;
-          break;
-        }
-      }
+      const inPrevKeyPressedArrIndex = this.prevKeyPressedArr.findIndex(address => address === charCode);
       if (inPrevKeyPressedArrIndex > -1) {
         this.prevKeyPressedArr.splice(inPrevKeyPressedArrIndex, 1);
       }
     }
     if (this.prevKeyPressedArr.length) {
-      isKeyDown = true;
-      charCode = this.prevKeyPressedArr[this.prevKeyPressedArr.length - 1];
+      isKeyDown = true; // eslint-disable-line
```

Comment:
I think it would be better to add what rule exactly do you ignore here and below

Reply:
Moreover almost any `eslint-disable...` should get removed from repo, but for speed purposes I had to use this short way as a temporary solution. The task for the rest is here:
https://jira.corp.ooyala.com/projects/PLAYER/issues/PLAYER-5215